internal/constants/constants.go

```diff
@@ -0,0 +1,7 @@
+package constants
+
+var BlacklistedCommands = []string{
```

Comment:
Let's rename this to `BlockedCommandsList`

Reply:
resolved

Reply:
It's `BlockedCommandsList` please fix it.